source/common/umf_pools/disjoint_pool.cpp

```diff
@@ -966,10 +966,10 @@ DisjointPool::~DisjointPool() {
     size_t HighPeakSlabsInUse;
     if (impl->getParams().PoolTrace > 1) {
         auto name = impl->getParams().name;
-        impl->printStats(TitlePrinted, HighBucketSize, HighPeakSlabsInUse,
-                         name.c_str());
-        if (TitlePrinted) {
-            try { // cannot throw in destructor
+        try { // cannot throw in destructor
+            impl->printStats(TitlePrinted, HighBucketSize, HighPeakSlabsInUse,
+                             name.c_str());
```

Comment:
Coverity is complaining about the `std::cout` possibly throwing [`std::ios_base::failure` ](https://en.cppreference.com/w/cpp/io/ios_base/failure). Practically this is pretty unlikely since something would have had to enable exceptions for stdout, but the `try catch` was already there (for the exact same reason I'm assuming), so expanding it seemed like an easy way to stop Coverity emitting this warning.